pkg/controller/operatingsystemconfig/actuator.go

```diff
@@ -220,6 +220,11 @@ func (a *actuator) configureNTPDaemon(extensionUnits []extensionsv1alpha1.Unit,
 			Content:     extensionsv1alpha1.FileContent{Inline: &extensionsv1alpha1.FileContentInline{Data: templateData}},
 			Permissions: ptr.To[uint32](0644),
 		})
+	case configv1alpha1.None:
+		extensionUnits = append(extensionUnits,
```

Comment:
Do we want to disable `systemd-timesyncd` and `ntpd` here? If we want to use something which is pre-configured in the image like PTP there is no need to disable anything. And there are maybe other cloud providers with other default flarcar images which uses `systemd-timesyncd` and `ntpd`.

Maybe `none` is misleading here and we should call it `keepDefault` or `unmanaged`.